tpm2/tpm2.go

```diff
@@ -216,79 +63,112 @@ func encodeShortPCRs(pcrNums []int) ([]byte, error) {
 	// For example, selecting PCRs 3 and 9 looks like:
 	// size(3)  mask     mask     mask
 	// 00000011 00000000 00000001 00000100
-	pcr := make([]byte, sizeOfPCRSelect+1)
-	pcr[0] = sizeOfPCRSelect
+	ts := tpmsPCRSelection{
+		Hash: sel.Hash,
+		Size: sizeOfPCRSelect,
+		PCRs: make([]byte, sizeOfPCRSelect),
+	}
 	// pcrNums parameter is indexes of PCRs, convert that to set bits.
```

Comment:
nit: I think this comment has gone stale, since there isn't a pcrNums parameter any more.

Reply:
Done